cfpb/owning-a-home
# Brand footer

Add brand footer to /explore-rates, /closing-disclosure, /loan-estimate, and /loan-options pages

## Additions

- Brand footer html, css, and images
- Instantiate brand footer on /explore-rates, /closing-disclosure, /loan-estimate, and /loan-options pages

## Removals

- Related links section on above pages


## Testing

- Check out branch and run grunt
- Check that brand footer appears on /explore-rates, /closing-disclosure, /loan-estimate, and /loan-options pages


## Review

- @higs4281 
- @sonnakim 

## Screenshots

explore-rates
<img width="1368" alt="screen shot 2016-11-18 at 11 12 00 am" src="https://cloud.githubusercontent.com/assets/778171/20446326/e286f79e-ad8d-11e6-990c-4b7cbb8a6836.png">


closing-disclosure
<img width="1363" alt="screen shot 2016-11-18 at 11 12 11 am" src="https://cloud.githubusercontent.com/assets/778171/20446297/c90c4760-ad8d-11e6-9e85-72dd529ee234.png">


loan-estimate
<img width="1363" alt="screen shot 2016-11-18 at 11 12 25 am" src="https://cloud.githubusercontent.com/assets/778171/20446265/a98a7bc8-ad8d-11e6-8076-c4f7c39c61a4.png">


loan-options
<img width="1365" alt="screen shot 2016-11-18 at 11 12 36 am" src="https://cloud.githubusercontent.com/assets/778171/20446190/4d891726-ad8d-11e6-936a-09b5c353d21e.png">


## Todos

- @sonnakim, I was wondering about breakpoints for the footer. Should there be a range between mobile & desktop where the sections stack 2x2?

## Checklist
